ntc_templates/templates/huawei_vrp_display_interface.textfsm

```diff
@@ -0,0 +1,73 @@
+Value Required INTERFACE (\S*)
+Value INTERFACE_STATUS (.*)
```

Comment:
```suggestion
Value LINK_STATUS (.*)
```

This one pains me, but for the consistency would like to move this to LINK_STATUS in order to remain consistent with the existing similar commands.